test/cli/test_cli_functions.sh

```diff
@@ -668,3 +668,37 @@ function check_backingstore {
     echo_time "💬  Deleting bucket testbucket"
     test_noobaa bucket delete "testbucket"
 }
+
+function check_dbdump {
+    # Generate db dump
+    echo_time "💬  Generaiting db dump"
+    test_noobaa db-dump --dir /tmp
+
+    # Check whether dump was created
+    dump=`ls -l /tmp | grep noobaa_db_dump | awk '{ print $9 }'`
+    if [ -z "$dump" ]
+    then
+        echo_time "❌  db dump was not generated"
+        exit 1
+    fi
+
+    # Remove dump file
+    rm /tmp/$dump
```

Comment:
based on the suggestion above: 
```suggestion
    rm $dump
```

Reply:
Did not modify per my comment on line 678